### PR TITLE
Implements CPack for easy RPM building

### PR DESCRIPTION
**Pull request by gabrielkay**
This adds the option to generate two rpms for this repository, one rpm with the necessary .so files and one devel rpm with necessary headers. Calling make rpm after a general make generates the rpms into .build/BUILD/RPMS.

### CODE REVIEW

**Comment by gabrielkay** [comment]
Can this one be merged?

**Comment by rnburn** [comment]
For RPMs, is it important to have them installed into lib64? The LIB_INSTALL_DIR option from opentracing-cpp wasn't ported over to this project yet, so they'd always be installed to lib.

File: CMakeLists.txt [hunk 1]
```diff
@@ -8,6 +9,30 @@ set(ZIPKIN_VERSION_PATCH "0")
...
+SET(CPACK_PACKAGE_DESCRIPTION_SUMMARY "C++ implementation of Zipkin")
+SET(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
+SET(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
+SET(CPACK_RPM_PACKAGE_GROUP "Development/Libraries")
+
+SET(CPACK_PACKAGE_VERSION_MAJOR ${ZIPKIN_VERSION_MAJOR})
+SET(CPACK_PACKAGE_VERSION_MINOR ${ZIPKIN_VERSION_MINOR})
+SET(CPACK_PACKAGE_VERSION_PATCH ${ZIPKIN_VERSION_PATCH})
+set(CPACK_RPM_DIST_POST_INSTALL_SCRIPT_FILE ${CMAKE_SOURCE_DIR}/scripts/runldconfig)
+set(CPACK_RPM_DIST_POST_UNINSTALL_SCRIPT_FILE ${CMAKE_SOURCE_DIR}/scripts/runldconfig)
```
Comment (rnburn) [inline CMakeLists.txt]: `runldconfig` is missing, no?

**Comment by gabrielkay** [comment]
Most RPMs are installed in lib64 but Installing in lib is fine, I set my program to also look in lib. The LIB_INSTALL_DIR variable would have to be added to do anything otherwise.

File: CMakeLists.txt (hunk 1 above)
Comment (gabrielkay) [reply CMakeLists.txt]: My local git lost track of it, I just pushed it here.

**Pull request merged**